### PR TITLE
Issue 2

### PR DESCRIPTION
**Pull request by siddhu001**
Try running in pipeline manner i.e. get the words as output from SLU and use them as input to your semantic parser

### CODE REVIEW

**Comment by siddhu001** [comment]
Log 1-
Use output of SLU as input to your semantic parser not gold set utterances
The embedding layer of semantic parser is initialised with the word classifier weights of SLU model

Validation loss - 0.8103000800757931, Validation Accuracy - 0.8046824887366231
Test loss - 0.22066965505908984, Test Accuracy -0.9343527550908528

**Review by viswavi — APPROVED** [review]
